### ISSUE TITLE
polybuild failing when page is including iron-component-page

### ISSUE THREAD

**Issue by deedubbu** [PolymerElements/iron-component-page#81]
I've ran into an issue with the `1.1.3` release of iron-component-page that is causing the polybuild of my project to fail with an error like this:

```
[15:59:44] Error in plugin "polyclean"
Message:
    Line 257: Unexpected token (
Details:
    index: 7362
    lineNumber: 257
    column: 27
    description: Unexpected token (
```

I've tracked it down to [this change](https://github.com/PolymerElements/iron-component-page/commit/37481d3defd132b194e7ff68d87dab7c54b7c2dd#diff-9f45ba230f9861b9b0b0c7417fa96561R477) in commit 37481d3defd132b194e7ff68d87dab7c54b7c2dd made in PR #76 . Not sure if it was intentional or not, but looks like an ES2015 method definition is being used. Uglifyjs apparently [does not support ES2015 yet](https://github.com/mishoo/UglifyJS2/issues/895). Simple fix would be to define the method without using ES2015 method definition.

```
_setActiveFromHash: function(hash) {
```

**Comment by notwaldorf** [comment]
Would you be interested in sending a PR to fix it? I think you're right, and it should indeed be `setActiveFromHash: function(hash)`.

**Comment by deedubbu** [comment]
It was fixed by #82. This can be closed.

**Comment by notwaldorf** [comment]
Yay, thanks!
